plugins/star-rating/api/api.js

```diff
@@ -110,14 +496,90 @@ var plugin = {},
                         });
                         common.returnOutput(params, result);
                         return true;
-                        //common.returnMessage(params, 200, result);
                     }
                 }
             );
             return true;
         }
         return false;
     });
+    
+    plugins.register("/i/apps/create", function(ob){
+        var params = ob.params;
+        var appId = ob.appId;
+        common.db.collection('feedback' + appId).ensureIndex({"uid":1}, function(){});
+        common.db.collection('feedback' + appId).ensureIndex({"ts":1}, function(){});
+    });
+
+    plugins.register("/i/apps/delete", function(ob){
+        var appId = ob.appId;
+        common.db.collection('web_feedback_widgets_' + appId).drop(function() {});
+        common.db.collection('feedback' + appId).drop(function() {});
+        common.db.collection("events"+crypto.createHash('sha1').update("[CLY]_star_rating" + appId).digest('hex')).drop(function() {});
+        if(common.drillDb)
+            common.drillDb.collection("drill_events" + crypto.createHash('sha1').update("[CLY]_star_rating" + appId).digest('hex')).drop(function() {});
+
+    });
+    
+    plugins.register("/i/apps/clear", function(ob){
+        var appId = ob.appId;
+        var ids = ob.ids;
+        common.db.collection('web_feedback_widgets_'+appId).remove({ts:{$lt:ob.moment.unix()}}, function() {});
```

Comment:
do we even have ts in that collection? As I understood it stores the feedback forms and not feedback itself, so it should only be deleted/cleared on app delete and app reset